src/services/affinidi/auth/index.ts

```diff
@@ -139,7 +139,7 @@ export class Auth implements AuthSDK {
 
     const projectScopedToken = await this.createProjectScopedToken(accessToken, activeProject.id)
     tokenService.setProjectToken(projectScopedToken, activeProject.id, activeProject.name)
-    tokenService.setPrincipalId(principalId)
+    tokenService.setPrincipal(principalId)
```

Comment:
Split the principal into Id and type before calling the service, maybe do it in the `receivePrincipalId`.
Does the whoAmI endpoint from IAM still return the principal ID merged with the type? If so let's notify Nucleus and request the change.

Reply:
done.